packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -126,25 +130,30 @@ export class SOQLEditorInstance {
     webviewPanel.onDidDispose(this.dispose, this, this.subscriptions);
   }
 
+  protected sendMessageToUi(
+    type: string,
+    payload?: string | string[] | DescribeSObjectResult
+  ): void {
+    this.webviewPanel.webview
+      .postMessage({
+        type,
+        payload
+      })
+      .then(undefined, async (err: string) => {
```

Comment:
it would be catastrophic for the postMessage to fail.  But now we'll know if it ever happens.